_templates/exchanges.html

```diff
@@ -13,6 +13,8 @@ <h2 class="exchanges-tab-title exchanges-international-title" id="international"
   <p>
     <a class="marketplace-link" href="https://www.bitstamp.net/">Bitstamp</a>
     <br>
+    <a class="marketplace-link" href="https://www.binance.com/">Binance</a>
```

Comment:
Upon further review, since there is no fiat deposit option, we're not sure this would be an appropriate fit for the International category - people would not have an easy way to exchange their fiat for bitcoin. This is a feature that all of the other exchanges in the International category have and could be very confusing to new users.